q2_moshpit/plugin_setup.py

```diff
@@ -255,3 +251,75 @@
                 'user inputs and uses those to construct a database.',
     citations=[citations["wood2019"]]
 )
+
+plugin.methods.register_function(
+    function=q2_moshpit.kraken2.select_kraken_features,
```

Comment:
I don't like the term "select" here... it sounds like some kind of selection is actually taking place. But this is more like reformatting or processing. `kraken_to_table`? `process_kraken`? `format_kraken`?